.sage/proto.go

```diff
@@ -15,7 +15,7 @@ const (
 	// Version of buf.build/googleapis/googleapis to generate from.
 	// Versions can be found here:
 	// https://buf.build/googleapis/googleapis/history
-	googleapisRef = "f3590c56d388417ebbedefae77ac12bf"
+	googleapisRef = "0487c63ac8292e507e0f3fecb3c01efdadea4967"
```

Comment:
update comment, github repo is used now

Reply:
On a second look it seems like a bad idea to use a specific ref in googleapis repository.
In that repository there are no tags or release branches, so must use a specific commit or master.

When using a specific commit the entire git history will eventually have to be cloned to check out the commit.

--

Will generate from master, and see if it is a problem.

Reply:
No, will generate specific commit with more fetch depth.